java.lang.AssertionError: Already disposed: Project (name=*, containerState=DISPOSE_COMPLETED, componentStore=C:\Users\rbabayev\IdeaProjects\*)

### Description



### Stacktrace
```
java.lang.AssertionError: Already disposed: Project (name=Multivariate Embeddings, containerState=DISPOSE_COMPLETED, componentStore=C:\Users\rbabayev\IdeaProjects\Multivariate Embeddings) 
	at com.intellij.openapi.components.impl.ComponentManagerImpl.lambda$throwAlreadyDisposed$0(ComponentManagerImpl.java:153)
	at com.intellij.openapi.application.ReadAction.lambda$run$1(ReadAction.java:62)
	at com.intellij.openapi.application.impl.ApplicationImpl.runReadAction(ApplicationImpl.java:825)
	at com.intellij.openapi.application.ReadAction.compute(ReadAction.java:69)
	at com.intellij.openapi.application.ReadAction.run(ReadAction.java:62)
	at com.intellij.openapi.components.impl.ComponentManagerImpl.throwAlreadyDisposed(ComponentManagerImpl.java:151)
	at com.intellij.openapi.components.impl.ComponentManagerImpl.getPicoContainer(ComponentManagerImpl.java:144)
	at com.intellij.serviceContainer.PlatformComponentManagerImpl.getService(PlatformComponentManagerImpl.kt:286)
	at com.intellij.openapi.components.ServiceManager.getService(ServiceManager.java:22)
	at com.intellij.execution.RunManager$Companion.getInstance(RunManager.kt:27)
	at nl.hannahsten.texifyidea.editor.autocompile.AutoCompileState$scheduleCompilation$1.invokeSuspend(AutoCompileState.kt:74)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(Dispatched.kt:241)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineSchedu

--- a/src/nl/hannahsten/texifyidea/editor/autocompile/AutoCompileState.kt
+++ b/src/nl/hannahsten/texifyidea/editor/autocompile/AutoCompileState.kt
@@ -70,6 +70,7 @@ object AutoCompileState {
         hasChanged = false
 
         // Get run configuration selected in the combobox and run that one
+        if (project!!.isDisposed) return
         val runConfigSettings = RunManager.getInstance(project!!).selectedConfiguration
 
         if (runConfigSettings?.configuration !is LatexRunConfiguration) {

